webrtc.html

```diff
@@ -6529,9 +6553,22 @@ <h2>Attributes</h2>
               <code><a>RTCRtpReceiver</a></code> object <var>receiver</var>.
               When one of the SSRCs for RTP source media streams received
               by <var>receiver</var> is removed (either due to reception of
-              a BYE or via timeout), the <code>mute</code> event is fired at
-              <code>track</code>.  If and when packets are received again,
-              the <code>unmute</code> event is fired at <code>track</code>.</p>
+              a BYE or via timeout), <a>set the muted state</a> of the
+              <code>track</code> to the value <code>true</code>.
+              Note that <a data-lt="set the RTCSessionDescription">
+              setting an <code>RTCSessionDescription</code></a>
+              can also lead to <a data-lt="set the muted state"> setting
+              the muted state</a> of the <code>track</code> to the
+              value <code>true</code>.
+              If packets are received again, and the <code>track</code> is
+              muted, and either the [[\PendingDirection]] or
+              [[\CurrentDirection]] slot of the 
```

Comment:
Is this 100% correct? Suppose `[[CurrentDirection]]` is `inactive`, and I receive a `sendrecv` remote offer. The remote endpoint isn't supposed to send anything until it receives my answer saying "yes, I agree to receiving." So if I receive packets in this state, shouldn't they be ignored?

This is different than the case where I *send* a `sendrecv` offer. Because in that case, it would be legal for media to arrive before the answer.

So, do we need to differentiate the cases where `[[PendingDirection]]` is set from a local offer vs. set from a remote offer? Or is my interpretation of the SDP direction rules wrong?

Reply:
As done now [[PendingDirection]] is _only_ used on the offering side, and its sole purpose is to (as you say) allow for media arriving before the answer to unmute.

We could perhaps expand to use [[PendingDirection]] on the answering side, but that's not part of this PR (and not needed for what it is trying to fix).

Reply:
I assume you mean "only on the local side" rather than "only on the offering side"? I guess this works then. Though it's not obvious from the name "PendingDirection," so maybe this could be clarified where it's defined.